tsconfig.json

```diff
@@ -1,11 +1,18 @@
 {
   "compilerOptions": {
     "target": "es6",
-    "module": "es2015",
+    "module": "commonjs",
```

Comment:
I guess we need that for node?

Reply:
Initially, it was throwing errors when I used `export`. This was the solution.

Reply:
I think node doesn't understand es6 modules yet. deno does but I'm not sure we want to switch to it yet

Reply:
I found [this link](http://wiki.commonjs.org/wiki/Modules/1.1.1) which explains how modules can be defined and used in `commonjs` format.

Reply:
Yeah, it's all gonna go away with es6 but that's okay. Keep this for now.

Reply:
@domoritz `commonjs` only right?

Reply:
@domoritz `import` in `es2015` isn't supported in Node. I think we will have to stick with commonjs. Reference: [https://stackoverflow.com/questions/39436322/node-js-syntaxerror-unexpected-token-import](https://stackoverflow.com/questions/39436322/node-js-syntaxerror-unexpected-token-import)